addons/editor-unshare-button/button.css

```diff
@@ -1,4 +1,4 @@
 .sa-unshare-button {
-  background: hsla(30, 100%, 55%, 1);
+  background: #ff4d4d;
```

Comment:
Changing the color of something from another addon should be discussed and committed separately.